samples/inputs/tooltip/advanced/package.json

```diff
@@ -24,6 +24,8 @@
     "babel-runtime": "^6.26.0",
     "core-js": "^3.6.5",
     "igniteui-webcomponents": "^6.3.1",
+    "igniteui-webcomponents-charts": "6.0.2-beta.5",
```

Comment:
According to NPM, the latest packages are 6.2.1:
https://www.npmjs.com/package/igniteui-webcomponents-charts
https://www.npmjs.com/package/igniteui-webcomponents-core